src/main/webapp/index.jsp

```diff
@@ -31,6 +31,7 @@
     <% } %>
     <a href="/about.jsp">About</a>
     <a href="/activity.jsp">Activity Feed</a>
+    <a href="profile.jsp">Profile</a>
```

Comment:
You probably want '/profile.jsp' instead of just 'profile.jsp' here.

Reply:
just fixed! Thank you